### ISSUE TITLE
new version?

### ISSUE THREAD

**Issue by deric** [rstacruz/ion#1]
Any chance to release `0.0.2`?

**Comment by rstacruz** [comment]
I can release your changes as 0.0.2 in a while (though there isn't any functionality changes!).

Sorry about this, but at this point this project is rather unmaintained. It served its purpose well as a proof-of-concept, but I strongly suggest you look at other solutions. If you want to help maintain this project, let me know.

**Comment by deric** [comment]
Right, still even as a proof of concept it looks quite ok. I was looking for some simple full text search on top of Redis, there's probably just one alternative:
- [redis-textsearch](https://github.com/nateware/redis-textsearch) - not really maintained as well

The code is quite messy, but they have inversed index, which is missing in Ion.

I like the way how you did design Ion, though there's a lot of space for improvement. I found few interesting links regarding full text search in Redis. I'd like to implement partial matching for strings, if I'm not mistaken this is not possible now, right?
- [Two ways of using Redis to build a NoSQL autocomplete search index](http://patshaughnessy.net/2011/11/29/two-ways-of-using-redis-to-build-a-nosql-autocomplete-search-index)
- [Auto Complete with Redis](http://oldblog.antirez.com/post/autocomplete-with-redis.html)
- [Redis Partial Word Match -- You (Auto)complete Me](https://robots.thoughtbot.com/redis-partial-word-match-you-auto-complete-me)

Except the last one, all of them are rather old and do not mention the [SSCAN](http://redis.io/commands/scan) command which (I think) could be effectively used for searching. I'll try to implement a proof of concept and I'll let you know.

Yeah, I'd like to help with maintaining the project.
